MetafileImporter/Logic/vtkSlicerMetafileImporterLogic.cxx

```diff
@@ -187,6 +187,41 @@ void UpdateTransformNodeFromString(vtkMRMLLinearTransformNode* transformNode, co
 }
 
 
+void UpdateStringFromTransformNode(std::string& str, vtkMRMLLinearTransformNode* transformNode)
```

Comment:
What about removing this method and instead add:

```
std::string toString(vtkMatrix4x4* mat) const;
```

in [Slicer/Libs/vtkAddon/vtkAddonMathUtilities.h](https://github.com/Slicer/Slicer/blob/master/Libs/vtkAddon/vtkAddonMathUtilities.h)